service/telemetry/generated_config.go

```diff
@@ -3,575 +3,116 @@
 package telemetry
 
 import (
-	"encoding/json"
-	"fmt"
+	"go.opentelemetry.io/contrib/config"
 )
 
-type Attributes struct {
-	// ServiceName corresponds to the JSON schema field "service.name".
-	ServiceName *string `mapstructure:"service.name,omitempty"`
-}
+// Deprecated: [v0.87.0] Use "go.opentelemetry.io/contrib/config".Attributes instead.
```

Comment:
version needs an update (in this file generally)
```suggestion
// Deprecated: [v0.90.0] Use "go.opentelemetry.io/contrib/config".Attributes instead.
```

Reply:
Fixed 👍🏻